### DISCUSSION TITLE
Add a note to `tw` props usage

### DISCUSSION THREAD

**Discussion by lamualfa** [ben-rogerson/twin.macro#490] [General] [answered]
the problem is that you are still need to import `twin.macro` in the file that use `tw` props. I was confused about why the use of `tw` props is not working because there's no error in console. and i only realized the problem when i matching again my code and the exisiting example code.

Please add some note into the example use of `tw` props in README.md about that needs.

Thanks.

**Comment by ben-rogerson** [comment, marked as answer]
Sure, this could be a good improvement.

Make sure you check out the [prop styling guide](https://github.com/ben-rogerson/twin.macro/blob/master/docs/prop-styling-guide.md#basic-styling) - you'll see there's a way to use the tw prop without the twin.macro import.
